public/documents/fari-rpgs/charge-srd.md

```diff
@@ -372,6 +372,21 @@ Once you've decided how many charges you want to consume, roll and interpret the
 - On a `6`, the `consequence` is avoided.
 - On a `66`, the `consequence` is avoided, and you get +1 position for your next roll.
 
+> #### Designer Note: Hacking Determination Rolls
+> 
+> Determination rolls act as a mechanism for the players to say "no, I don't think so" when something bad happens to their character.
+> 
+> There are two important things you should look at if you want to hack this mechanic. 
+> 
+> - First, how does one resist a consequence. 
+> - Second, what should be the cost of trying to resist a consequence. 
+> 
+> By default, you try to resist by spending momentum to build a determination pool in the hope of reducing or even canceling an incoming consequence. The cost, in the case, is paid in `momentum` since you will not be able to spend later to `push yourself` or `assist` an ally.
```

Comment:
```suggestion
> By default, you try to resist by spending momentum to build a determination pool in the hope of reducing or even canceling an incoming consequence. In this case, the cost is paid in `momentum` since you won't be able to spend more later to `push yourself` or `assist` an ally.
```